syntax = "proto3";

message FindGatewayResponse {
  string ip = 1;
  int32 port = 2;
}